src/Omnimail.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Omnimail;
+
+use Omnimail\Common\Factory;
+
+/**
+ * Omnimail class
+ * This class was inspired by / copied from the Omnipay library.
+ *
+ * Provides static access to the gateway factory methods.  This is the
+ * recommended route for creation and establishment of mail gateway
+ * objects via the standard Factory.
+ *
+ * Example:
+ *
+ * <code>
+ *   // Create a gateway for the Silverpop mail client
+ *   // (routes to Factory::create)
+ *   $mailer= Omnimail::create('Silverpop');
+ *
+ *   // Get the mailer parameters.
+ *   $parameters = $mailer->getParameters();
+ * </code>
+ *
+ * @method static array all()
+ * @method static array replace(array $mailers)
+ * @method static string register(string $className)
+ * @method static array find()
+ * @method static array getSupportedMailers()
+ * @method static EmailSenderInterface create(string $class, array $parameters = [])
+ */
+class Omnimail
+{
+    const AMAZON_SES = '\Omnimail\AmazonSES';
```

Comment:
So this bit worries me. Does this mean every provider needs to be added here then if someone wants to add a new mailer then you have to approve the change - and you could wind up with a lot of review work for stuff that you could just leave to fester in the wild, or mature. 

I guess it's OK in that without declaring it Omnimail::create('AmasonSES'); should still work based on the file being findable?